src/main/java/com/additional/library/domain/Exemplar.java

```diff
@@ -18,15 +18,16 @@
     private ExemplarStatus status;
     private Book book;
 
-    public Exemplar(ExemplarStatus status) {
+    public Exemplar(ExemplarStatus status, Book book) {
```

Comment:
czy nie jest tak, że jak tworzymy nowy egzemplarz, to status powinniśmy ustawić na AVAILABLE domślnie? :)